docs/modules/ROOT/pages/advanced-config.adoc

```diff
@@ -74,9 +74,32 @@ spec:
 [TIP]
 .Passing environment variables
 ====
-If you want to pass environment variables to the `backupCommand` you'll have to wrap them in a shell. Example:
+If you want to pass environment variables to the `backupCommand` you'll have to wrap them in a shell. In above example that would look like this:
 
-`kubectl -n wordpress-demo annotate pods ${YOUR_POD_NAME} "appuio.ch/backupcommand=/bin/bash -c 'mysqldump -uroot -p\"\${MARIADB_ROOT_PASSWORD}\" --all-databases'" --overwrite`
+[source]
+--
+spec:
+  backupCommand: /bin/bash -c 'mysqldump -uroot -p "${MARIADB_ROOT_PASSWORD}" --all-databases'" --overwrite
+--
+
+You can also add it to any pods that are running in the namespace:
+
+[source]
+--
+via kubectl:
+kubectl -n wordpress-demo annotate pods ${YOUR_POD_NAME} "appuio.ch/backupcommand=/bin/bash -c 'mysqldump -uroot -p\"\${MARIADB_ROOT_PASSWORD}\" --all-databases'" --overwrite
+
+in the manifest:
+spec:
+  serviceName: "mysql"
```

Comment:
```suggestion
  serviceName: "mariadb"
```
Because the environment variable also talks about mariadb

Reply:
Right changing